### ISSUE TITLE
marytts-component-installer.sh +x

### ISSUE THREAD

**Issue by lucasb-eyer** [strands-project/strands_ui#53]
In the packaged version, the file `marytts-5.0/bin/marytts-component-installer.sh` should be installed with executable flags:

```
strands@karl:~$ /opt/ros/hydro/share/mary_tts/marytts-5.0/bin/marytts-component-installer.sh
bash: /opt/ros/hydro/share/mary_tts/marytts-5.0/bin/marytts-component-installer.sh: Permission denied
strands@karl:~$ ll /opt/ros/hydro/share/mary_tts/marytts-5.0/bin/marytts-component-installer.sh
-rw-r--r-- 1 root root 188 Nov 12 11:52 /opt/ros/hydro/share/mary_tts/marytts-5.0/bin/marytts-component-installer.sh
```

**Comment by cdondrup** [comment]
Since I'm working on mary atm any way, I will fix that.

**Comment by cdondrup** [comment]
Closed via #54
